src/main/scala/uk/gov/nationalarchives/tdr/api/http/GraphQLServer.scala

```diff
@@ -110,3 +115,5 @@ object GraphQLServer {
       }
   }
 }
+
+class GraphQLServer
```

Comment:
This is new - is it needed?

Reply:
If we want to use the `classOf` for the logger as GraphQLServer is an object. Otherwise will need to hard code the value for the Logger: `Logger("...GraphQLServer")`

Reply:
Was just getting the `classOf` for the logger to work. Removed it